src/cgame/cg_main.cpp

```diff
@@ -480,17 +356,11 @@ void CG_UpdateBuildableRangeMarkerMask()
 		{ "none",           0                                       },
 	};
 
-	if ( cg_rangeMarkerBuildableTypes.modificationCount != btmc ||
-	     cg_rangeMarkerWhenSpectating.modificationCount != spmc )
+	if ( Util::optional<std::string> structureList = cg_rangeMarkerBuildableTypes.GetModifiedValue() )
 	{
-		int brmMask = cg_rangeMarkerWhenSpectating.integer ? ( 1 << BA_NONE ) : 0;
+		int brmMask = 0;
 
-		if ( !cg_rangeMarkerBuildableTypes.string[ 0 ] )
-		{
-			goto empty;
-		}
-
-		for (Parse_WordListSplitter marker(cg_rangeMarkerBuildableTypes.string); *marker; ++marker)
+		for (Parse_WordListSplitter marker(*std::move(cg_rangeMarkerBuildableTypes)); *marker; ++marker)
```

Comment:
I'm not sure this is correct: how can you move the cvar? How is this different from 
```suggestion
		for (Parse_WordListSplitter marker(std::move(cg_rangeMarkerBuildableTypes.Get())); *marker; ++marker)
```

Reply:
Good catch, that was supposed to say `structureList`.